packages/daemon/package.json

```diff
@@ -57,9 +57,10 @@
     "ethereumjs-tx": "^1.3.7",
     "ethereumjs-util": "^6.0.0",
     "ethereumjs-wallet": "^0.6.2",
-    "fs-extra": "^7.0.0",
+    "fs-extra": "^7.0.1",
     "graphql": "^0.13.2",
     "graphql-relay": "^0.5.5",
+    "keythereum": "^1.0.4",
```

Comment:
Do we need to add this dependency to the daemon please? There doesn't seem to be any code change in the daemon using it.